### DISCUSSION TITLE
Restricting repository names with a ruleset in a GitHub Enterprise Cloud organization

### DISCUSSION THREAD

**Discussion by bmitc** [community/community#143979] [Enterprise]
### Select Topic Area

Question

### Body

Similar to restricting branch names across all repositories in a GitHub Enterprise Cloud organization, I want to restrict the repository names themselves.

For example, I have the following branch ruleset:

![image](https://github.com/user-attachments/assets/d892d9bc-35ff-45b8-8a35-12b70e559ad9)

And then inside that ruleset, I use a regular expression to restrict branch names:

![image](https://github.com/user-attachments/assets/61074751-ba19-497b-8219-f1df1afc1ba8)

I want to do the exact same thing but for repository names and am unable to find a way to do this.

1. Is this possible?
2. If so, how?

**Comment by bew** [comment]
I have the same usecase, we want to enforce a set of prefix for all repositories, to be able to immediately distinguish the repos of various teams that use the same organization.
